test/e2e/framework/helpers/common/utils.go

```diff
@@ -102,7 +103,11 @@ func RandSeq(n int) string {
 	var letters = []rune("abcdefghijklmnopqrstuvwxyz0123456789")
 	b := make([]rune, n)
 	for i := range b {
-		b[i] = letters[rand.Intn(len(letters))]
```

Comment:
please add the ignore as we don't need crypt quality

Reply:
No problem.